index.html

```diff
@@ -0,0 +1,416 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Youtubish</title>
+  </head>
+  <body>
+    <div class="top-menu">
```

Comment:
Consider using semantic html (in this case `<header>` for this section)

Reply:
Please ensure you include an explanation, a "why", for each requested change. For example:
This is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a `div`, that would improve the HTML structure and make the page more accessible*

Reply:
Excellent, your comment is polite and straightforward.